src/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -37,6 +40,7 @@ internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
 		private readonly ILogEventSender m_logsEventSender;
 		private readonly IExternalScopeProvider m_defaultExternalScopeProvider;
 		private readonly IEnumerable<ILogScrubbingRule> m_textScrubbers;
+		private readonly IOptions<OmexLoggingOptions> m_options;
```

Comment:
```suggestion
		private readonly OmexLoggingOptions m_options;
```

Reply:
I dont think this work because OmexLoggerProvider is created by DI, and IOption is the way to inject it. We cannot inject OmexLoggingOptions instead

Reply:
This is not injecting OmexLoggerProvider.
It would be injecting IOptions<OmexLoggerProvider> options.
Then the class member would be OmexLoggingOptions settings, which would be assigned options.Value in the construction.